instrumentation/opentelemetry-instrumentation-flask/src/opentelemetry/instrumentation/flask/__init__.py

```diff
@@ -397,7 +397,10 @@ def _before_request():
 
         activation = trace.use_span(span, end_on_exit=True)
         activation.__enter__()  # pylint: disable=E1101
-        flask_request_environ[_ENVIRON_ACTIVATION_KEY] = activation
+        flask_request_environ[_ENVIRON_ACTIVATION_KEY] = (
+            get_ident(),
```

Comment:
Consider adding a new environ key for the thread_id, It looks like the thread_id is not part of the activation

Reply:
Oh. That's a really good point. Let me update the PR.